crates/re_arrow_store/src/store_format.rs

```diff
@@ -123,8 +126,9 @@ impl std::fmt::Display for IndexedBucket {
 
         let time_range = {
             let time_range = &self.inner.read().time_range;
+            // TODO(paris): Understand if this is correct or if we should use show_timestamps_in_local_timezone. Same with other store_* files.
```

Comment:
For displaying a raw bucket directly via the display trait this is fine.
Generally I think UTC is a good default for the Display trait where we naturally can't take user preferences into account.
In the long run more and more ui should be more specific about its needs calling dedicated methods which is opposed to "dump this data into a string for human consumption". 
Implies ofc that `std::fmt::Display` soon won't do the trick in a lot of situations.

Reply:
Great thanks, leaving in UTC.